draft-ietf-quic-qlog-quic-events.md

```diff
@@ -1522,17 +1517,34 @@ KeyType =
 
 ## QUIC Frames
 
+The generic `$QuicFrame` is defined here as a CDDL extension point (a "socket"
+or "plug"). This allows other documents to extend this extension point when
+defining new QUIC protocol frame types to enable automated validation of
+aggregated qlog schemas.
```

Comment:
The intent of the sentence isn't that clear to me. Who is wanting to automate validation, does somebody producing qlog or consuming qlog care?

How about simplifying to :

```suggestion
or "plug"). It can be extended to support additional QUIC frame types.
```